### DISCUSSION TITLE
[PG] Doesn't show other information in table DDL (grants)

### DISCUSSION THREAD

**Discussion by Cavaldier** [dbeaver/dbeaver#9072] [Q&A] [answered]
7.1.1 PostgreSQL
If i go to any table i can't see any grants on table. It will be helpful if that will came back(in version 6~ it was implemented but in 7 i don't have that. Maybe, i just need check some checkbox. Please help

**Comment by uslss** [comment, marked as answer]
Hi @Cavaldier ,
probably "Show permissions" button is unchecked
![image](https://user-images.githubusercontent.com/31996417/85411199-b4a68d00-b570-11ea-912f-785989727fb9.png)
BR

**Comment by Cavaldier** [comment]
Thanks :D you are big :D thanks a lot

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Wolny
od wirusów. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

wt., 23 cze 2020 o 15:44 uslss <notifications@github.com> napisał(a):
